Formula/aws-sam-cli-nightly.rb

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+require_relative '../ConfigProvider/config_provider'
+
+class AwsSamCliNightly < Formula
+  include Language::Python::Virtualenv
+
+  config_provider = ConfigProvider.new('aws-sam-cli-nightly')
+
+  desc "AWS SAM CLI 🐿 is a tool for local development and testing of Serverless applications"
+  homepage "https://github.com/aws/aws-sam-cli/"
+  url config_provider.url()
+  sha256 config_provider.sha256
+  head "https://github.com/aws/aws-sam-cli.git", :branch => "nightly-builds"
+
+  bottle do
+    root_url config_provider.root_url()
+    cellar :any_skip_relocation
+    sha256 config_provider.sierra_hash() => :sierra
+    sha256 config_provider.linux_hash() => :x86_64_linux
+  end
+
+  depends_on "python@3.8"
+
+  def install
+    venv = virtualenv_create(libexec, "python3.8")
+    system libexec/"bin/pip", "install", "--upgrade", "pip"
+    system libexec/"bin/pip", "install", "-v", "--pre", "--ignore-installed", buildpath
+    system libexec/"bin/pip", "uninstall", "-y", "aws-sam-cli"
```

Comment:
Should this line be `system libexec/"bin/pip", "uninstall", "-y", "aws-sam-cli-nightly"`?

Reply:
I think here we do the uninstallation to make sure there is no conflict with aws-sam-cli. I saw aws-sam-cli-rc also uninstalled the main package: https://github.com/aws/homebrew-tap/blob/f8aeb5034a95beb70f8cc3e938c9276fd813bad9/Formula/aws-sam-cli-rc.rb#L30

Reply:
But this one is not conflicting with `aws-sam-cli` anymore, can we verify if we still need this?

Reply:
I guess we are still sharing pypi with normal sam cli.
Test removed this line, the symlink doesn't create.